src/expenses.router.js

```diff
@@ -0,0 +1,85 @@
+const { Router } = require('express');
+const service = require('./expensesServices.js');
+const expensesRouter = Router();
+
+expensesRouter.get('/', async (req, res) => {
+  const expenses = await service.getAllExpenses(req.query);
+
+  res.status(200).json(expenses);
+});
+
+expensesRouter.post('/', async (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+
+  if (!userId || !spentAt || !title || !amount || !category || !note) {
+    return res.sendStatus(400);
+  }
+
+  const expense = await service.addExpense(
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  );
+
+  if (!expense) {
+    return res.sendStatus(400);
+  }
```

Comment:
```suggestion

```
Consider wrapping the logic into the try-catch block and sending the 500 error if the expense is not created and there is an error

Reply:
okay, but anyway I have to keep this code, cause it's needed for tests